src/vm.ts

```diff
@@ -269,6 +269,16 @@ function step(n: number = 0): void {
   runSheet.getRange(`${apColumn}${n + 2 + 1}`).setValue(newAp);
 }
 
+function run() {
+  let i: number = 0;
```

Comment:
I'd like this function to start from the latest step instead, so that you can start by making few steps with `Step` and then `Run` to run until the end

Reply:
updated the i to start at the latest pcColumn index so that it starts from the latest step.